es6-shim.js

```diff
@@ -205,7 +205,15 @@
     return result;
   };
 
+  var bindContext = function (func, context) {
+    return function () {
+      return func.apply(context, arguments);
+    };
+  };
+
   var ES = {
+    Call: bindContext(Function.prototype.call, Function.prototype.apply),
```

Comment:
I think that `ES.Call` could be really useful, and if it can be written as `Function.call.bind(Function.apply)`, I think `bindContext` might as well be removed.